config/routes.rb

```diff
@@ -9,11 +10,13 @@
   root 'users#index'
```

Comment:
- [ ] According to what the routes file shows, you are repeating some routes which cause problems while creating a DB. I recommend removing some repeated routes. For example, you used `root 'users#index'` twice (line 4 and line 10).